bootstrap/secret/secure.go

```diff
@@ -477,3 +480,19 @@ func (p *SecureProvider) GetSelfJWT() (string, error) {
 func (p *SecureProvider) IsJWTValid(jwt string) (bool, error) {
 	return p.secretClient.IsJWTValid(jwt)
 }
+
+func (p *SecureProvider) HttpTransport() http.RoundTripper {
+	return p.httpRoundTripper
+}
+
+func (p *SecureProvider) SetHttpTransport(rt http.RoundTripper) {
+	p.httpRoundTripper = rt
+}
+
+func (p *SecureProvider) IsZeroTrustEnabled() bool {
+	return p.zeroTrustEnabled
+}
+
+func (p *SecureProvider) ZeroTrustEnabled() {
```

Comment:
Match how define in the interface.

```suggestion
func (p *SecureProvider) EnableZeroTrust() {
```

Reply:
This changes wasn't made